### ISSUE TITLE
api.symfony.com: invalid certificate + 502 error

### ISSUE THREAD

**Issue by Martin-P** [symfony/symfony-docs#16322]
The Symfony documentation contains links to api.symfony.com, e.g. the section [Debugging Emails](https://symfony.com/doc/current/mailer.html#debugging-emails):
> The [SentMessage](https://api.symfony.com/6.0/Symfony/Component/Mailer/SentMessage.html) object returned by the `send()` method of the [TransportInterface](https://api.symfony.com/6.0/Symfony/Component/Mailer/Transport/TransportInterface.html) provides access to [...]

This domain however contains an invalid certificate. The certificate is rejected by the browser because the used certificate does not belong to api.symfony.com, but is registered for `*.fr-4.platformsh.site`. Furthermore when I allow the browser to continue by accepting this wrong certificate there is a 502 error displayed:

> Error 502
> No route known for api.symfony.com 

Basically, it is broken in multiple ways.

Looking at Google cache it looks like it was mirroring the Symfony repository from GitHub.

**Comment by fabpot** [comment]
I think it's time to not use it anymore and use Github links directly.

**Comment by wouterj** [comment]
Agreed, see https://github.com/symfony-tools/docs-builder/pull/136

**Comment by javiereguiluz** [comment]
This was fixed by Wouter in the Pull Request mentioned above. We'll see the fix in production after the docs are rebuilt tonight and Cloudflare cache expires after that. Closing as fixed then. Thanks!

**Comment by Martin-P** [comment]
Works like a charm again.Thanks for the fix!
